napari/_tests/test_config.py

```diff
@@ -0,0 +1,490 @@
+import os
+import stat
+import sys
+import shutil
+import tempfile
+from collections import OrderedDict
+from contextlib import contextmanager
+
+import pytest
+
+import napari.config  # noqa
+from napari.config import (
+    canonical_name,
+    collect,
+    collect_env,
+    collect_yaml,
+    config,
+    ensure_file,
+    expand_environment_variables,
+    get,
+    merge,
+    refresh,
+    rename,
+    set,
+    update,
+    update_defaults,
+)
+
+yaml = pytest.importorskip("yaml")
```

Comment:
Also here...

Reply:
removed, since we depend on yaml.  But that's a nice pytest trick that lets you try to import something, but then just skip any tests that use the module if it's not available